app/assets/stylesheets/db/common/base.scss

```diff
@@ -0,0 +1,31 @@
+body {
+  padding-top: 70px;
+}
+
+nav.navbar {
+  height: 60px;
+  line-height: 60px;
+
+  .navbar-header .navbar-brand {
+    height: 60px;
+    line-height: 60px;
+    padding: 0 15px;
+  }
+
+  .navbar-collapse {
+    .navbar-nav a {
+      line-height: 60px;
+      padding: 0 15px;
+    }
+
+    form.navbar-form {
+      height: 60px;
+      line-height: 56px;
+      margin: 0;
+
+      input.form-control {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 3, but was 4